drivers/char/random.c

```diff
@@ -871,8 +871,18 @@ void __init random_init_early(const char *command_line)
 	/* Reseed if already seeded by earlier phases. */
 	if (crng_ready())
 		crng_reseed(NULL);
-	else if (trust_cpu)
-		_credit_init_bits(arch_bits);
+	else if (trust_cpu) {
+
+		// _credit_init_bits(arch_bits);
+		
+		/*
+		 * With random.trust_cpu=on in the kernel command line, 
+   		 * sets entropy to 256 to initialize random number geenrator
+      		 * in early boot.
+		 */
+		size_t artifical_arch_bits = 256;
```

Comment:
Why did you choose 256? What is the threshold?

Reply:
256 is the threshold, lemme comment that